### PR TITLE
Introduce job for aggregating topics

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: f8a_jobs/handlers/aggregate_topics.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        if from_date is not None:
+            from_date = parse_datetime(from_date)
+        if to_date is not None:
+            to_date = parse_datetime(to_date)
+
+        s3 = StoragePool.get_connected_storage('S3Data')
+        # TODO: this will need to be changed once we will introduce package level flows
+        postgres = StoragePool.get_connected_storage('BayesianPostgres')
+
+        base_query = self.postgres.session.query(WorkerResult).\
```
Comment (jpopelka) [inline f8a_jobs/handlers/aggregate_topics.py]: self.postgres -> postgres  ?

Comment (fridex) [inline f8a_jobs/handlers/aggregate_topics.py]: Good point, fixed. Thanks.

File: f8a_jobs/handlers/aggregate_topics.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        :param to_date: date limitation for taks result queries
+        """
+        if from_date is not None:
+            from_date = parse_datetime(from_date)
+        if to_date is not None:
+            to_date = parse_datetime(to_date)
+
+        s3 = StoragePool.get_connected_storage('S3Data')
+        # TODO: this will need to be changed once we will introduce package level flows
+        postgres = StoragePool.get_connected_storage('BayesianPostgres')
```
Comment (jpopelka) [inline f8a_jobs/handlers/aggregate_topics.py]: Please do since https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/27 has been merged

Comment (fridex) [inline f8a_jobs/handlers/aggregate_topics.py]: Huh, this is tricky - this was requested as @miteshvp wants to proceed with his work. Currently we have all data in `BayesianPostgres`, so we will need first to create a one-time job to move these data do `PackagePostgres`.

So what about merging this and once we ingest more packages, we will switch the old `github_details` to appropriate table/s3 bucket?

Comment (jpopelka) [inline f8a_jobs/handlers/aggregate_topics.py]: Agree

**Pull request merged**